js/packages/cli/src/commands/upload.ts

```diff
@@ -811,36 +811,36 @@ export async function upload({
         ),
       );
     }
-
-    const {
-      properties: { creators },
-      seller_fee_basis_points: sellerFeeBasisPoints,
-      symbol,
-    } = getAssetManifest(dirname, '0');
-
-    const config = cache.program.config
-      ? new PublicKey(cache.program.config)
-      : await initConfig(anchorProgram, walletKeyPair, {
-          totalNFTs,
-          mutable,
-          retainAuthority,
-          sellerFeeBasisPoints,
-          symbol,
-          creators,
-          env,
-          cache,
-          cacheName,
-        });
-
-    setAuthority(walletKeyPair.publicKey, cache, cacheName, env);
-
-    return writeIndices({
-      anchorProgram,
-      cache,
-      cacheName,
-      env,
-      config,
-      walletKeyPair,
-    });
   }
+
+  const {
+    properties: { creators },
+    seller_fee_basis_points: sellerFeeBasisPoints,
+    symbol,
+  } = getAssetManifest(dirname, '0');
+
+  const config = cache.program.config
+    ? new PublicKey(cache.program.config)
+    : await initConfig(anchorProgram, walletKeyPair, {
+        totalNFTs,
+        mutable,
+        retainAuthority,
+        sellerFeeBasisPoints,
+        symbol,
+        creators,
+        env,
+        cache,
+        cacheName,
+      });
+
+  setAuthority(walletKeyPair.publicKey, cache, cacheName, env);
+
+  return writeIndices({
+    anchorProgram,
+    cache,
+    cacheName,
+    env,
+    config,
+    walletKeyPair,
+  });
 }
```

Comment:
What's the actual change above?  It's a bit unclear, seems like a cope/paste of the same code, but github is showing all lines are changed — did you convert tabs to spaces, or something similar? Maybe not seeing something.

Reply:
I moved one of the close curly brackets "}" up / moved some blocks of code down, outside of an if statement so that the upload to config account process is not skipped when all items have been uploaded.

Reply:
As I understand, this block of code was moved out the "if (dedupedAssetKeys.length)" body (it starts on line 700 in the commit 4f835f73e632ccaf4eb913d8fb64518ff52eb237).
Indent was changed from 4 spaces to 2, and the curly bracket moved from line 845 to 814.